<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Адресная книга</string>
  </property>
  <property name="windowIcon">
   <iconset resource="files.qrc">
    <normaloff>:/images/contacts.png</normaloff>:/images/contacts.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="contact_list_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>391</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>240</width>
      <height>0</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="contact_list_layout">
       <item>
        <widget class="QLineEdit" name="filter_edit">
         <property name="toolTip">
          <string>Фильтр контактов</string>
         </property>
         <property name="placeholderText">
          <string>Введите поисковой запрос</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="contact_list">
         <property name="minimumSize">
          <size>
           <width>220</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="remove_contact_layout">
         <item>
          <widget class="QPushButton" name="add_contact_btn">
           <property name="toolTip">
            <string>Добавить контакт</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="files.qrc">
             <normaloff>:/images/add_in_db.png</normaloff>:/images/add_in_db.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="remove_contact_btn">
           <property name="toolTip">
            <string>Удалить контакт</string>
           </property>
           <property name="text">
            <string>Удалить</string>
           </property>
           <property name="icon">
            <iconset resource="files.qrc">
             <normaloff>:/images/delete.ico</normaloff>:/images/delete.ico</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="contact_info_widget" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>301</width>
      <height>381</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>240</width>
      <height>0</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="contact_info_layout">
       <item>
        <layout class="QHBoxLayout" name="name_image_layout">
         <item>
          <widget class="QToolButton" name="image_btn">
           <property name="minimumSize">
            <size>
             <width>110</width>
             <height>110</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="files.qrc">
             <normaloff>:/images/dummy.png</normaloff>:/images/dummy.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="name_edit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="other_layout">
         <item row="0" column="0">
          <widget class="QLabel" name="private_number_lbl">
           <property name="text">
            <string>Личный телефон</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="private_number_edit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="office_number_lbl">
           <property name="text">
            <string>Рабочий телефон</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="office_number_edit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="address_lbl">
           <property name="text">
            <string>Адрес</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="address_edit"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="email_lbl">
           <property name="text">
            <string>Почта</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="email_edit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="comment_lbl">
           <property name="text">
            <string>Комментарий</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="comment_edit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSpinBox" name="id">
         <property name="maximum">
          <number>999999</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="ready_btn_layout">
         <item>
          <widget class="QPushButton" name="refresh_btn">
           <property name="text">
            <string>Обновить</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="ready_btn">
           <property name="toolTip">
            <string>Сохранить контакт</string>
           </property>
           <property name="text">
            <string>Готово</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="add_contact_action"/>
    <addaction name="separator"/>
    <addaction name="quit_action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="add_contact_action">
   <property name="icon">
    <iconset resource="files.qrc">
     <normaloff>:/images/add_in_db.png</normaloff>:/images/add_in_db.png</iconset>
   </property>
   <property name="text">
    <string>Новый контакт</string>
   </property>
  </action>
  <action name="quit_action">
   <property name="icon">
    <iconset resource="files.qrc">
     <normaloff>:/images/quit.png</normaloff>:/images/quit.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="files.qrc"/>
 </resources>
 <connections/>
</ui>
